geojson.opam.template

```diff
@@ -0,0 +1,3 @@
+pin-depends: [
+  ["ocaml-geojson" "git+https://github.com/geocaml/ocaml-geojson"]
```

Comment:
The package name is `geojson` not `ocaml-geojson`. I think it also requires a version, so you can use `.dev`

Reply:
geojson.dev

Reply:
The file name needs to be `topojson.opam.template`